queries/ispc/highlights.scm

```diff
@@ -0,0 +1,225 @@
+; inherits: c
+
+[
+  "soa"
+  "task"
+  "launch"
+  "unmasked"
+  "template"
+  "typename"
+  (sync_expression)
+] @keyword
+
+[
+  "in"
+  "new"
+  "delete"
+] @keyword.operator
+
+[
+  "cdo"
+  "cfor"
+  "cwhile"
+  "foreach"
+  "foreach_tiled"
+  "foreach_active"
+  "foreach_unique"
+] @repeat
+
+[
+  "cif"
+] @conditional
+
+[
+  "varying"
+  "uniform"
+] @type.qualifier
+
+"__regcall" @attribute
```

Comment:
```suggestion
"__regcall" @storageclass
```

Reply:
The `__regcall` attribute does not change the storage class of an object, it affects function argument passing in registers (https://www.intel.com/content/www/us/en/docs/cpp-compiler/developer-guide-reference/2021-9/c-c-calling-conventions.html).  A similar attribute is `__vectorcall` which is already handled in `c/highlights.scm` as `@attribute`.  I would treat `__regcall` the same as `__vectorcall`.

Reply:
I was thinking of storageclass in a loose sense that a type of calling convention can be linked to its layout/storage at runtime, but I didn't see other similar ones already marked as attribute...seems fine then

Reply:
Since Intel C/C++ compilers support `__regcall` it should probably be moved into `c/highlights.scm` anyway.